packages/web/localizations/en.json

```diff
@@ -22,7 +22,7 @@
     "step2Title": "Step 2: Add Liquidity",
     "step2TitleManaged": "Step 2: Select Provider",
     "managed": "Managed",
-    "managedDescription": "Enlist one of our providers to manage your position. An entirely hands off experience.",
+    "managedDescription": "Enlist a community liquidity manager to actively manage your position through Magma vaults.",
```

Comment:
BTW this update isn't reflected in other languages. 

It's somewhat tedious to update since we need to remove all this entry from all files, add it to en.json, then run `yarn machine-translate` to generate updated ones in all files.

But this may not be high prio enough to do.